integration_tests/e2e/prisons/prisons.cy.ts

```diff
@@ -7,18 +8,31 @@ context('Supported prisons', () => {
     cy.task('reset')
     cy.task('stubSignIn')
     cy.task('stubAuthUser')
+    cy.task('stubPrisons')
+    cy.task('stubGetAllPrisons')
```

Comment:
I think it is ok for `stubPrisons` because this is the call to prison register that might be called on any page at any time depending on cache state. Feels like it's 'setting the scene' like `stubAuthUser` etc.

I *think* `stubGetAllPrisons` is only used in the listing page test so agree that if that is the case it's best there.

Reply:
Actually - `stubGetAllPrisons` probably needed for both tests so perhaps OK - but to keep an eye on if we refactor.